src/components/elements/Inputs/FileInput/FileInput.tsx

```diff
@@ -44,6 +52,9 @@ const FileInput = (props: FileInputProps) => {
 
   const ref = useRef<HTMLInputElement>(null);
 
+  // eslint-disable-next-line no-unused-vars
```

Comment:
I'm curious why this is needed? It looks like it is used; maybe it's from an early draft of this change?

Reply:
line deleted